.github/workflows/actions.yml

```diff
@@ -4,12 +4,43 @@ on: [push]
 
 jobs:
   tests:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
why do you need this change?

Reply:
I think we should have separate actions for the hammer part. can you separate all hammer related things into another action pipeline?